lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/JavaBridge.java

```diff
@@ -18,6 +18,9 @@
  */
 package org.apache.reef.javabridge;
 
+/**
+ * TODO[JIRA REEF-383] Document/Refactor JavaBridge.
```

Comment:
Any items which are not "TODO[JIRA ...]" are reported as warnings. REEF-519 is to remove all such warnings, so yes, it will fix TODO[REEF...] as well (just in that case triage will be easier :-) ).